apl-integration-tests/src/test/java/com/apollocurrrency/aplwallet/inttest/model/TestBase.java

```diff
@@ -48,9 +46,15 @@ static void initAll() {
         restHelper = new RestHelper();
         ClassLoader classLoader = TestBase.class.getClassLoader();
         String secretFilePath = Objects.requireNonNull(classLoader.getResource("APL-MK35-9X23-YQ5E-8QBKH")).getPath();
-        importSecretFileSetUp(secretFilePath,"1");
-        startForgingSetUp();
-        setUpTestData();
+        try {
+            importSecretFileSetUp(secretFilePath,"1");
+            startForgingSetUp();
+            setUpTestData();
+        }catch (Exception ex){
+            Assert.assertNotNull("Precondition FAILED", ex);
```

Comment:
Looks like such statement makes no sense

Reply:
This is done specifically for display in the report.